scripts/servisi.coffee

```diff
@@ -155,6 +155,24 @@ module.exports = (bot) ->
                 r.reply "Vrednost BTC v #{currency.toUpperCase()}: last: #{data.last}, low: #{data.low}, high: #{data.high}, bid: #{data.bid}, ask: #{data.ask}"
               else
                 r.reply "Bitstamp is down"
+          else
+              r.reply "Nau šlo. Poskusi .btc eur ali .btc usd."
+
+    bot.regexp /^\.eth (.+)/,
+      ".eth <valuta> - izpiše trenutno ETH vrednost na coinmarketcap.com",
+        (match,r) ->
+          currency = match[1].toLowerCase()
+          if currency == "eur" || currency == "usd"
+            bot.fetchJSON "https://api.coinmarketcap.com/v1/ticker/ethereum/?convert=EUR", (data) ->
+              if data && ! null
```

Comment:
! Null ?

Reply:
Don't kill me for copy/pasting existing code!
https://github.com/ubuntu-si/ircbot/blob/master/scripts/servisi.coffee#L154 :smiley: